packages/webapp/pages/account/notifications.tsx

```diff
@@ -274,7 +274,7 @@ const AccountNotificationsPage = (): ReactElement => {
       });
     }
 
-    subscribePersonalizedDigest({ type, hour: preferredHour });
+    subscribePersonalizedDigest({ type, hour: preferredHour , sendType : personalizedDigestType as SendType  });
```

Comment:
To fix the lint issue.

```suggestion
    subscribePersonalizedDigest({
      type,
      hour: preferredHour,
      sendType: personalizedDigestType as SendType,
    });
```

Reply:
Linter wants it all on one line